internal/utils/utils_test.go

```diff
@@ -48,6 +48,14 @@ func TestGenKey(t *testing.T) {
 	GenKey("3.3.3.3", "3456")
 }
 
+// Test for GenKeyBLS
+func TestGenKeyBLS(t *testing.T) {
+	priKey, pubKey := GenKeyBLS("3.3.3.3", "3456")
+	if priKey == nil || pubKey == nil {
```

Comment:
can you find a way to compare the key generated is correct or not?

Reply:
There is not much special logic in the function. It's basically putting what is generated from GetUniqueIDFromIPPort as bytes into private key and call the GetPublicKey util of BLS. I just add result checking for it based on the output. But we are trusting the BLS package is working fine for now.